{"name": "test_failed_two", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "def test_failed_two():\n>       assert False\nE       assert False\n\ntests/hooks_example_two/test_hooks_two.py:10: AssertionError"}, "start": 1708622330447, "stop": 1708622330447, "uuid": "ff3a4834-3c3e-482f-a8fc-679fe01d5c7d", "historyId": "f06e2a9e3dd12432c135ff9ce70a8cd6", "testCaseId": "6d40bfc4e1d11b93f29dc3326bfbe44b", "fullName": "tests.hooks_example_two.test_hooks_two#test_failed_two", "labels": [{"name": "parentSuite", "value": "tests.hooks_example_two"}, {"name": "suite", "value": "test_hooks_two"}, {"name": "host", "value": "Macbook-Pro-Flora.local"}, {"name": "thread", "value": "21814-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.hooks_example_two.test_hooks_two"}]}